packages/kit-bg/src/dbs/local/LocalDbBase.ts

```
@@ -2749,18 +2728,101 @@ export abstract class LocalDbBase extends LocalDbBaseContainer {
     });
   }
 
-  async getAllIndexedAccounts() {
+  async getAllDevices(): Promise<{ devices: IDBDevice[] }> {
+    const cacheKey = 'allDbDevices';
+    const allDevicesInCache = this.dbAllRecordsCache.get(
+      cacheKey,
+    ) as IDBDevice[];
+    if (allDevicesInCache && allDevicesInCache.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)